module_utils/network/junos/config/base.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+The base class for all junos resource modules
+"""
+
+from ansible.module_utils.connection import Connection
+
+
+class ConfigBase(object):  # pylint: disable=R0903
```

Comment:
This needs to be refactored as per the updated changes in ansible/ansible and RMB builder code